stable/hazelcast/templates/statefulset.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1beta1
```

Comment:
Do you need to support K8s < 1.9? Otherwise, please use `apps/v1`.

Reply:
Done
We don't need K8 < 1.9.